lib/commands/gesture.js

```diff
@@ -9,6 +9,29 @@ let helpers = {}, extensions = {}, commands = {};
 
 commands.moveTo = iosCommands.gesture.moveTo;
 
+
+function requireFloatParameter (paramName, paramValue, methodName) {
+  if (_.isUndefined(paramValue)) {
```

Comment:
This check (and the one in `requireIntParameter`) seems unnecessary. `parseFloat(undefined)` returns `NaN`, so it would be caught in the `isNaN` check.

Reply:
I want to show different error messages for missing and incorrect parameter values